webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesNavigationContent.vue

```diff
@@ -162,7 +164,7 @@ export default {
   },
   methods: {
     searchSpaces() {
-      return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/${eXo.env.portal.containerName}/social/spaces/lastVisitedSpace/list.json?offset=${this.offset}&limit=${this.limitToFetch}`, {
+      return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spaces?q=&offset=${this.offset}&limit=${this.limitToFetch}&filterType=lastVisited&returnSize=true&expand=member,favorite`, {
```

Comment:
When changing the default REST URLs retrieved by a portlet, please make sure to not degrade the performances of page loading by updating the `preloaded` fetched URLs configured in `portlet.xml`. for this concrete example, the modification should go here : https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/WEB-INF/portlet.xml#L403 

@SaraBoutej @AzmiTouil FYI.

Reply:
Thank you for your review, well noted (y), Will fix it !